test/src/tests-identity.ts

```diff
@@ -642,6 +642,36 @@ describe('identity', function() {
         });
     });
 
+    describe('privacy flags', function() {
+        beforeEach(function() {
+            mParticle._resetForTests(MPConfig);
+            mParticle.config.flags.cacheIdentity = 'True';
+            localStorage.clear();
+        });
+
+        describe('#createIdentityCache', function() {
+            it('should use LocalStorageVault when noFunctional is false by default', async () => {
```

Comment:
you aren't really confirming that the vault is or isn't localstorage vs disablevault.